deploy-service/common/src/main/resources/sql/deploy.sql

```diff
@@ -472,3 +472,9 @@ CREATE TABLE IF NOT EXISTS pas_configs (
     pas_state VARCHAR(32) NOT NULL,
     PRIMARY KEY (group_name)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS group_mappings (
+     asg_group_name    VARCHAR(64) NOT NULL,
+     cluster_name VARCHAR(64) NOT NULL,
```

Comment:
I saw you use VARCHAR(128) for spot_auto_scaling_groups table. Could you please make group_mappings table columns to VARCHAR(128)?